helm-v3/006-repositories.md

```diff
@@ -17,20 +29,25 @@ and could benefit from a capability to push packages from the Helm client.
 
 To support registries there will be a `helm push` command that works in a
 similar manner to `helm fetch`. It will support different systems via the
-URI/URL scheme and provide a default implementation for HTTP(S). Other systems,
-such as S3, can implement an uploader via a plugin for schemes Helm does not
+URI/URL scheme. It will provide a default implementation for HTTP(S) based
+on the ChartMuseum API spec.
+
+Other systems, such as S3, can implement an uploader via a plugin for schemes Helm does not
 provide an implementation for.
 
-> [name=Matt Farina] There very well may be additional work here around
-> registry search and pluggable or otherwise extendable authenication
-> mechanisms.
-> 
-> [name=Adnan Abdulhussein] What would the default HTTP(S) implementation be?
-> [name=Matt Fisher] Adnan, would you mind clarifying your comment a little?
-> 
-> [name=Adnan Abdulhussein] Sorry, the proposal mentions there will be a
-> default implemention for `helm push` to a regular HTTP(S) chart repository.
-> It's unclear to me how this would actually work.
-> 
-> [name=Adnan Abdulhussein] Sounds like this is something that will be fleshed
-> out more in Josh's proposal, so I'll wait for that :)
+The [helm-push plugin](https://github.com/chartmuseum/helm-push) describes potential semantics
```

Comment:
I like that ChartMuseum is already moving on the push work. This document on repositories is what we're doing rather than potential work. Could the docs here be crafted for how we plan to implement it rather than potential?

If we're not sure how it will work that we could put a WIP on the title and plan some time to work out the details. We need this and to nail down the first draft of how it will work.